edn/src/types.rs

```diff
@@ -24,7 +26,7 @@ pub enum Value {
     Float(OrderedFloat<f64>),
     Text(String),
     Symbol(String),
-    Keyword(String),
+    Keyword(keyword::Keyword),
```

Comment:
Yeah, I really think we want `impl From<Into<String>> for keyword::Keyword ...` if that's legal, or `From<String>` if not.  Then we can still construct `Value::Keyword` the easy way.